IBF-Typhoon-model/lib_r/damage_probability.R

```diff
@@ -0,0 +1,85 @@
+# Functions for calculating the damage probabilities
+
+
+get_total_impact_forecast <- function(df_impact_forecast, damage_thresholds, organization){
+  
+  # Get the total summarized impact
+  # For some reason if <- is used here the method can't find this var
+  df_total_impact_forecast = df_impact_forecast %>%
+    group_by(GEN_typhoon_name, GEN_typhoon_id) %>%
+    dplyr::summarise(CDamaged_houses = sum(Damaged_houses)) %>%
+    group_by(GEN_typhoon_name) %>%
+    dplyr::summarise(
+      # TODO: Is there a more elegant way to do this?
```

Comment:
I ended up making a [post on StackOverflow](https://stackoverflow.com/questions/68819628/iterating-over-values-and-variable-names-in-dplyrsummarise/68820572#68820572), where they suggested to use `purrr:map_dfc`. I think it's good to generalize, because that way you can pass a different number of thresholds / probabilities and it will still work.